MockPyth.sol

```diff
@@ -8,14 +8,24 @@ contract MockPyth is AbstractPyth {
     mapping(bytes32 => PythStructs.PriceFeed) priceFeeds;
     uint64 sequenceNumber;
 
+    uint singleUpdateFeeInWei;
```

Comment:
should this be configurable without a contract upgrade?

something to think about in future PRs, no need to change this one.

Reply:
It's only the mock contract. I don't think consumers that use this for testing actually need this. 

In the real contract it has an update method (for owner).